CRM/Contact/Page/AJAX.php

```diff
@@ -676,14 +676,14 @@ public static function getDedupes() {
 
     $nextParamKey = 3;
     $mappings = array(
-      'src' => 'cc1.display_name',
```

Comment:
Minor point, and not an issue caused by this PR so we can tackle it later. Using 'src' and 'dst' isn't consistent with the other terminology we use. We should either use 'src' and 'dst' everywhere, or change this to 'main' and 'other' to help preserve what's left of our sanity.